Tools/MiniBrowserSwiftUI/Configurations/Base.xcconfig

```diff
@@ -23,7 +23,7 @@
 // THE POSSIBILITY OF SUCH DAMAGE.
 //
 
-SDKROOT = macosx.internal
+SDKROOT = macosx
```

Comment:
I don't think that we want to make this change for all builds. Please see other projects (e.g. `Tools/MiniBrowser/Configurations/DebugRelease.xcconfig`) for how to make `SDKROOT` conditional.